src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAudioRequestIntTest.java

```diff
@@ -34,7 +34,7 @@
 @ActiveProfiles({"intTest", "h2db"})
 @AutoConfigureMockMvc
 @Slf4j
-class AudioControllerAddAudioRequestIntTest extends IntegrationBase {
+class AudioControllerAudioRequestIntTest extends IntegrationBase {
```

Comment:
Can this filename be kept as before please - it will probably end up being renamed to be `AudioRequestsController...IntTest` when it gets moved to the `AudioRequestsApi` later on.

Reply:
ACCEPT.